src/main/java/no/nav/familie/integrasjoner/journalpost/internal/SafJournalpostRequest.kt

```diff
@@ -13,6 +16,14 @@ data class SafRequest(
     val antall: Int = 200,
 )
 
+data class SafRequestForBruker(
+    val brukerId: Bruker,
+    val antall: Int,
+    val tema: List<Tema>?,
+    val journalposttype: List<Journalposttype>?,
+    val journalstatus: List<String>? = emptyList(), // Dersom denne ikke blir sendt med så filtrerer SAF på journalposter som er ferdigstilte
```

Comment:
Kan man ikke unngå hele denne endringen hvis man kun endrer `JournalposterForBrukerRequest` i kontrakter med `val journalstatus: List<String> = emptyList()` (notere at jeg ikke har med nullable, då det gir uønsket funksjonalitet då man kun får med ferdigstilte)
Og oppdaterer integrasjoner med kontrakter? 

Då støtter man også filtrering av statuser direkt, ut mot klientene (selv om de faktiskt ikke har behov for det)

Reply:
Hvis ikke så tenker jeg at man burde rulle tilbake endringen i `journalposterForBruker.graphql` ang `journalstatuser` fra https://github.com/navikt/familie-integrasjoner/pull/848/files
Då man like gjerne kan ha en tom liste direkte der, i stedet for å mappe det til et nytt requestobjekt?

Reply:
Vi snakket om å endre JournalposterForBrukerRequest først, men når vi ikke mapper dette til noe i integrasjoner føles det litt vaklende. Dersom noen skulle finne på å endre JournalposterForBrukerRequest (kanskje brukerstatuser er bedre enn brukerstatus er vi like langt.

Reply:
Litt vel tett knytning opp mot saf sin graphql å bare ha et kontraktsobjekt som skal "fungere" føler jeg. Her er det faktisk et poeng å mappe om slik at objektet mellom ef/ks/ba og integrasjoner en dag kan endres uten å knekke alt.

Og vi trenger jo å filtrere på statuser via andre kall, så da hadde vi isåfall måtte lage TO graphql-query-filer.